### PR TITLE
Promo banner for encryption campaign

### LINKED ISSUE #1619: (Push Live on Feb 16) Implement Encryption Campaign promo banner for teach site
Based on - Design Encryption Campaign promo banner for teach site #1535 

Comp: https://redpen.io/p/ya4ae604c136854f8e

Assets (both SVG and PNG version): https://drive.google.com/a/mozilla.com/folderview?id=0B2irLVDJivhLU0FwS1I5N2hKdEU&usp=sharing

ai: https://drive.google.com/a/mozilla.com/file/d/0B2irLVDJivhLNnlXQmdFSjI1UmM/view?usp=sharing

Links to mzl.la/encrypt (which I think is https://advocacy.mozilla.org/encryption but will need to confirm). Also need to confirm launch date, which I think is either the 8th or 10th?

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #1619 

- :exclamation: ~~https://mzl.la/encrypt links https://advocacy.mozilla.org/ (not https://advocacy.mozilla.org/encryption as what's mentioned in #1619 )~~
- :exclamation: ~~need to confirm the launch date of Encryption campaign~~ (we are pushing this live on Feb 16)

### CODE REVIEW

File: img/pages/home/svg/encrypt.svg [hunk 1]
```diff
@@ -0,0 +1,311 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (Pomax) [inline img/pages/home/svg/encrypt.svg]: can we can clean up this SVG a bit?

File: img/pages/home/svg/encrypt.svg [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+<svg width="430px" height="75px" viewBox="0 0 430 75" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.5.1 (25234) - http://www.bohemiancoding.com/sketch -->
+    <title>Page 1</title>
+    <desc>Created with Sketch.</desc>
+    <defs>
+        <path id="path-1" d="M0,0.026 L430,0.026 L430,75 L0,75"></path>
```
Comment (Pomax) [inline img/pages/home/svg/encrypt.svg]: path-1, -3, -5, -7, -9, -11 and -13 are all the same.

File: img/pages/home/svg/encrypt.svg [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+            <path d="M116.293,52.6293 C114.958,52.6293 113.876,53.7123 113.876,55.0473 C113.876,56.3833 114.958,57.4653 116.293,57.4653 C117.628,57.4653 118.71,56.3833 118.71,55.0473 C118.71,53.7123 117.628,52.6293 116.293,52.6293" id="Fill-226" fill="#5380B7" sketch:type="MSShapeGroup"></path>
+            <path d="M116.293,61.3965 C114.958,61.3965 113.876,62.4795 113.876,63.8145 C113.876,65.1505 114.958,66.2325 116.293,66.2325 C117.628,66.2325 118.71,65.1505 118.71,63.8145 C118.71,62.4795 117.628,61.3965 116.293,61.3965" id="Fill-228" fill="#5380B7" sketch:type="MSShapeGroup"></path>
+            <path d="M116.293,70.1638 C114.958,70.1638 113.876,71.2468 113.876,72.5818 C113.876,73.9178 114.958,74.9998 116.293,74.9998 C117.628,74.9998 118.71,73.9178 118.71,72.5818 C118.71,71.2468 117.628,70.1638 116.293,70.1638" id="Fill-230" fill="#5380B7" sketch:type="MSShapeGroup"></path>
+            <mask id="mask-16" sketch:name="Clip 233" fill="white">
+                <use xlink:href="#path-15"></use>
+            </mask>
+            <g id="Clip-233"></g>
```
Comment (Pomax) [inline img/pages/home/svg/encrypt.svg]: good job sketch! groups without any content!

File: img/pages/home/svg/encrypt.svg [hunk 4]
```diff
@@ -0,0 +1,311 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="430px" height="75px" viewBox="0 0 430 75" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.5.1 (25234) - http://www.bohemiancoding.com/sketch -->
```
Comment (Pomax) [inline img/pages/home/svg/encrypt.svg]: this comment, the title, and desc, are all things that don't matter for our purpose

**Comment by mmmavis** [comment]
SVG updated! :smile:

**Comment by Pomax** [comment]
R+, although the placement feels a little forced (it doesn't feel like a "part of the page" section for some reason)

**Comment by mmmavis** [comment]
double checked just now: https://mzl.la/encrypt points to https://advocacy.mozilla.org/encrypt/ :dancers:

**Pull request merged**